chapters/synchronous.tex

```diff
@@ -1118,44 +1118,45 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 However, even if relations do not generate events, when-clauses and operators \lstinline!edge! and \lstinline!change! should behave as normal.
 \end{nonnormative}
 
-From the view of the clocked partition, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
-Therefore, such a partition is handled in the same way as any other clocked partition.
-Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized continuous-time partition with other partitions.
-Hereby, a discretized continuous-time partition is seen as a clocked partition.
+From the view of other partitions, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
+Therefore, such a partition is externally treated as any other clocked partition.
```

Comment:
This would be more clear to me (there is just a single continuous-time partition):
```suggestion
Therefore, outside the continuous-time partition itself, it is treated similar to clocked and discretized partitions.
```

Reply:
Good - **except for the next comment**. I have reformulated the entire section to be clearer.

Reply:
OK.  Let's resolve this and look at the new section instead.